### PR TITLE
Bump Play to 2.5

### PR DESCRIPTION
**Pull request by jonathonherbert**
## What does this change?

Bumps Play to 2.5.

## How to test

Run locally, or deploy to CODE. Does the app behave as expected?

### CODE REVIEW

File: conf/logback.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
         </encoder>
     </appender>
 
-    <root level="INFO">
+    <root level="DEBUG">
```
Comment (andrew-nowak) [inline conf/logback.xml:18]: should we revert this for a prod deploy?

**Comment by andrew-nowak** [comment]
unsure if I'm using it correctly, but on CODE I can neither create nor edit any packages (any POST xhrs get a 403 response)

**Comment by jonathonherbert** [comment]
Hasty request for review there, we're mobbing on this now. I'll demote to draft.

File: project/plugins.sbt [hunk 2]
```diff
@@ -10,12 +10,12 @@ resolvers ++= Seq(
...
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.19")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.0.3")
 
 addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.1.12")
 
-
 addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
 
+libraryDependencies += "org.vafer" % "jdeb" % "1.3" artifacts (Artifact("jdeb", "jar", "jar"))
```
Comment (alinaboghiu) [inline project/plugins.sbt:21]: how were we building a deb package before? or is this a new thing?

**Review by alinaboghiu — APPROVED** [review]

**Review by rebecca-thompson — APPROVED** [review]

File: project/plugins.sbt (hunk 2 above)
Comment (jonathonherbert) [reply project/plugins.sbt:21]: It's new as of sbt-native-packager 1.1.0, and a bit weird – in [this change to sbt-native-packager](https://github.com/sbt/sbt-native-packager/releases/tag/v1.1.0), the sbt plugin marks jdeb as a `provided` dependency, which IIUC means sbt assumes its available on the classpath. So we must provide it explicitly.

**Pull request merged**